docs/_static/nav-logo-microsoft-white.svg

```diff
@@ -0,0 +1,52 @@
+			c0.63,0,1.25,0.21,1.83,0.64l0.08,0.06V7.26l-0.03-0.01C48.25,7.13,47.96,7.02,47.59,6.95z"/>
+		<path fill="#FFFFFF" d="M53.91,6.89c-0.48,0-0.91,0.15-1.28,0.46c-0.32,0.27-0.56,0.64-0.74,1.09h-0.02V7.03h-1.92v8.16h1.92
+			v-4.17c0-0.71,0.16-1.29,0.48-1.73c0.31-0.44,0.73-0.66,1.24-0.66c0.17,0,0.37,0.03,0.58,0.08c0.21,0.06,0.36,0.12,0.45,0.18
+			l0.08,0.06V7.01L54.66,7C54.48,6.93,54.23,6.89,53.91,6.89z"/>
+		<path fill="#FFFFFF" d="M59.12,6.83c-1.35,0-2.41,0.39-3.18,1.17c-0.76,0.78-1.15,1.85-1.15,3.2c0,1.28,0.38,2.3,1.12,3.05
+			c0.74,0.75,1.75,1.13,3.01,1.13c1.3,0,2.35-0.4,3.11-1.19c0.76-0.79,1.15-1.85,1.15-3.17c0-1.3-0.36-2.33-1.08-3.08
+			C61.39,7.21,60.39,6.83,59.12,6.83z M60.65,13.11c-0.36,0.45-0.9,0.68-1.61,0.68c-0.7,0-1.26-0.23-1.65-0.69
+			c-0.39-0.46-0.59-1.12-0.59-1.96c0-0.86,0.2-1.54,0.59-2.01c0.39-0.47,0.94-0.71,1.64-0.71c0.67,0,1.21,0.23,1.59,0.67
+			c0.39,0.45,0.58,1.12,0.58,2C61.2,11.98,61.01,12.66,60.65,13.11z"/>
+		<path fill="#FFFFFF" d="M67.44,10.4c-0.6-0.24-0.99-0.44-1.15-0.6c-0.15-0.15-0.23-0.36-0.23-0.63c0-0.24,0.1-0.43,0.3-0.58
+			c0.2-0.15,0.48-0.23,0.83-0.23c0.32,0,0.66,0.05,0.98,0.15c0.33,0.1,0.62,0.23,0.86,0.4l0.08,0.05V7.19l-0.03-0.01
+			C68.85,7.08,68.56,7,68.2,6.93c-0.36-0.07-0.68-0.1-0.96-0.1c-0.92,0-1.67,0.23-2.25,0.7c-0.58,0.46-0.88,1.07-0.88,1.81
+			c0,0.38,0.06,0.72,0.19,1.01c0.13,0.29,0.32,0.54,0.58,0.76c0.26,0.21,0.65,0.43,1.18,0.66c0.44,0.18,0.77,0.34,0.98,0.46
+			c0.2,0.12,0.35,0.24,0.43,0.36c0.08,0.11,0.12,0.27,0.12,0.46c0,0.55-0.41,0.81-1.26,0.81c-0.31,0-0.67-0.07-1.06-0.19
+			c-0.39-0.13-0.76-0.31-1.09-0.55l-0.08-0.06v1.87l0.03,0.01c0.28,0.13,0.62,0.23,1.03,0.32c0.41,0.08,0.78,0.13,1.1,0.13
+			c0.99,0,1.79-0.24,2.38-0.7c0.59-0.47,0.88-1.09,0.88-1.85c0-0.55-0.16-1.02-0.48-1.4C68.75,11.05,68.2,10.71,67.44,10.4z"/>
+		<path fill="#FFFFFF" d="M74.7,6.83c-1.35,0-2.41,0.39-3.18,1.17c-0.76,0.78-1.15,1.85-1.15,3.2c0,1.28,0.38,2.3,1.12,3.05
+			c0.74,0.75,1.75,1.13,3.01,1.13c1.3,0,2.35-0.4,3.11-1.19c0.76-0.79,1.15-1.85,1.15-3.17c0-1.3-0.36-2.33-1.08-3.08
+			C76.97,7.21,75.96,6.83,74.7,6.83z M76.22,13.11c-0.36,0.45-0.9,0.68-1.61,0.68c-0.7,0-1.26-0.23-1.65-0.69
+			c-0.39-0.46-0.59-1.12-0.59-1.96c0-0.86,0.2-1.54,0.59-2.01c0.39-0.47,0.94-0.71,1.64-0.71c0.67,0,1.21,0.23,1.59,0.67
+			c0.39,0.45,0.58,1.12,0.58,2C76.77,11.98,76.59,12.66,76.22,13.11z"/>
+		<path fill="#FFFFFF" d="M88.99,8.59V7.03h-1.94V4.6l-0.07,0.02l-1.83,0.56l-0.04,0.01v1.84h-2.88V6c0-0.48,0.11-0.84,0.32-1.09
+			c0.21-0.24,0.51-0.36,0.89-0.36c0.27,0,0.56,0.06,0.85,0.19l0.07,0.03V3.13l-0.03-0.01c-0.27-0.1-0.63-0.14-1.08-0.14
+			c-0.57,0-1.08,0.12-1.53,0.37c-0.45,0.25-0.8,0.59-1.05,1.04C80.43,4.82,80.3,5.33,80.3,5.9v1.13h-1.35v1.57h1.35v6.59h1.94V8.59
+			h2.88v4.19c0,1.73,0.81,2.6,2.42,2.6c0.26,0,0.54-0.03,0.82-0.09c0.29-0.06,0.49-0.12,0.6-0.19l0.03-0.01v-1.58l-0.08,0.05
+			c-0.11,0.07-0.24,0.13-0.39,0.17c-0.15,0.04-0.28,0.07-0.38,0.07c-0.38,0-0.65-0.1-0.83-0.3c-0.17-0.2-0.26-0.55-0.26-1.05V8.59
+			H88.99z"/>
```

Comment:
:disappointed_relieved: good catch, can't believe I missed that... so embarrassing :blush: